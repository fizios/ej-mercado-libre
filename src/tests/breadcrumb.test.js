import React from "react";
import renderer from 'react-test-renderer';

import Breadcrumb from '../components/breadcrumb';

import listMock from './mockJson/list.json';

const categoriesMock = listMock.categories;

const element = (
    <Breadcrumb categories={categoriesMock} />
);

describe('[ BREADCRUMB ]', () => {
  it('Renders and it matches snapshot', () => {
    const wrapper = renderer.create(element);
    expect(wrapper).toMatchSnapshot();
  });
});
