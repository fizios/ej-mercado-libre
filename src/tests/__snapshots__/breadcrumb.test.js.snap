// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ BREADCRUMB ] Renders and it matches snapshot 1`] = `
<div
  className="breadcrumb"
>
  <span
    className="category-item"
  >
    Electr√≥nica, Audio y Video
     &gt; 
  </span>
  <span
    className="category-item"
  >
    Audio
     &gt; 
  </span>
  <span
    className="category-item"
  >
    Radios AM/FM
    
  </span>
</div>
`;
